master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE checkpoints_v2 
```

Comment:
I think in the past we have had migration files that are just empty when we can't go back

I'm not sure if the file not existing would cause any problems, but I think I would prefer to keep the assumption that each migration has a downfile true even if empty

example of us doing a blank migration file
https://github.com/determined-ai/determined/blob/main/master/static/migrations/20210917133742_drop-workload-start-time-and-id.tx.down.sql